CMakeLists.txt

```diff
@@ -28,9 +25,11 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 project(chimera-sdk LANGUAGES C ASM)
 
 include(${CMAKE_CURRENT_LIST_DIR}/cmake/Utils.cmake)
+include(${CMAKE_CURRENT_LIST_DIR}/cmake/opentitan.cmake)
 
-add_subdirectory(targets)
+add_subdirectory(drivers)
 add_subdirectory(hal)
+add_subdirectory(targets)
```

Comment:
`targets` should IMO be imported first, unless there is a good reason to change the convention.

Reply:
Yes totally agree. I think the PR is not yet rebased, and thus the changes are shown in a weird way.